
<script >
    document.title = "OmicsDI Help";
</script>

        <div id="content" class="row" style="margin: 10px">
            <div id="" class="col-md-9 col-lg-9" role="main"  >
                <h1 id="Overview">
                    Overview
                </h1>


                <p class="align-justify">
                    The Omics Discovery Index (OmicsDI) provides dataset discovery across a heterogeneous,
                    distributed group of genomics, proteomics and metabolomics data resources spanning eight
                    repositories in two continents and six organisations, including both open and controlled access data
                    resources. The resource provides a short description of every dataset: accession, description,
                    sample/data protocols biological evidences, publication, etc. The search capabilities offer a unique
                    resource to search for omics datasets. This fact converts the OmicsDI in the first resource worldwide to
                    provide search capabilities through multi-omics experiments.
                </p>


                <h1 id="MajorPartner">
                    Major Partners
                </h1>


                <h2 id="ProteomeXchange">
                    <a href="http://www.proteomexchange.org/">ProteomeXchange</a>
                </h2>


                <p class="align-justify">
                    The ProteomeXchange Consortium is a collaboration of currently three major mass spectrometry
                    proteomics data repositories, <a href="http://www.ebi.ac.uk/pride/archive/">PRIDE</a> at EMBL-EBI in
                    Cambridge Â (UK), <a href="http://peptideatlas.org">PeptideAtlas </a> at ISB in Seattle (US), and <a
                        href="http://massive.ucsd.edu"> MASSive</a> at UCSD (US), offering a unified data deposition and
                    discovery strategy across all three repositories. ProteomeXchange is a distributed database
                    infrastructure; the potentially very large raw data component of the data is only held at the
                    original submission database, while the searchable metadata is centrally collected and indexed. All
                    ProteomeXchange data is fully open after release of the associated publication.
                </p>


                <h2 id="MetabolomeXchange">
                    <a href="http://metabolomexchange.org/">MetabolomeXchange</a>
                </h2>


                <p class="align-justify">
                    MetabolomeXchange is a collaboration of 4 major metabolomics repositories, with a total of 10
                    partners contributing. MetabolomeXchange was inspired by and is implementing similar coordination
                    strategies to ProteomeXchange. The founding partners are MetaboLights at <a
                        href="http://www.ebi.ac.uk/metabolights/"> EMBL-EBI(UK)</a>, <a
                        href="http://www.cbib.u-bordeaux2.fr/">Metabolomics Repository Bordeaux</a>(FR), <a
                        href="http://gmd.mpimp-golm.mpg.de/">Golm Metabolome Database </a> and the <a
                        href="http://www.metabolomicsworkbench.org/">Metabolomics Workbench</a> (US). The Metabolomics
                    Workbench is a NIH funded collaboration of 6 Regional Comprehensive Metabolomics Resource Cores.
                    MetabolomeXchange started accepting metadata submissions in summer 2014, and reached 200 public
                    datasets in March 2015.
                </p>


                <h2 id="EGA">
                    The European Genome-Phenome Archive
                </h2>


                <p class="align-justify">
                    The European Genome-Phenome Archive (EGA) provides a service for the permanent archiving and
                    distribution of personally identifiable genetic and phenotypic data resulting from biomedical
                    research projects. Data at EGA was collected from individuals whose consent agreements authorise
                    data release only for specific research use to bona fide researchers. Strict protocols govern how
                    information is managed, stored and distributed by the EGA project. The EGA comprises a public
                    metadata section, allowing searching and identifying relevant studies, and the controlled access
                    data section. Access to the data section for a particular study is only granted after validation of
                    a research proposal through the relevant ethics approval.
                </p>


                <h2 id="CurrentDatabases">
                    Current Databases
                </h2>


                <p class="align-justify">
                    The original OmicsDI project provides access to eleven different databases with proteomics, genomics, transcriptomics and
                    metabolomics data from Europe and Unite States (see <a
                        href="#/databases">updated list </a>). The original list includes:
                    PRIDE (proteomics, UK), PeptideAtlas (proteomics, US), MassIVE (proteomics, US), GPMDB (proteomics, Canada), Metaboligths
                    (metabolomics, UK), MetabolomeWorkbench (metabolomics, US), GNPS (metabolomics, USA), MetabolomeExpress (metabolomics, Australia),
                    ArrayExpress and GEO (transcriptomics, genomics, UK), ExpressionAtlas (UK), EGA (genomics, UK).
                </p>


                <p class="align-justify">
                    The project is open for new partners and databases (please contact us: <a
                        href="omicsdi-support@ebi.ac.uk">omicsdi-support@ebi.ac.uk</a>). If you are interested in the
                    architecture and the metadata that the resource should provide see section <a href="https://github.com/BD2K-DDI/specifications#23-Omicsdi-architecture">2.3</a> and <a href="https://github.com/BD2K-DDI/specifications#24-omicsDI-xml">2.4</a> in "Specifications" of OmicsDI.
                </p>


                <h1 id="OmicsDIwebapplication">
                    OmicsDI web application
                </h1>


                <p class="align-justify">
                    The main goal of OmicsDI project is to have a way to search interesting datasets across omics
                    repositories. The main web application and web service (see <a
                        href="#/api">here</a>) allow the user to search and
                    navigate through the OmicsDI datasets. The OmicsDI web application offers two main access modes: (i) using the home page navigation panels or (ii) the search box.
                </p>


                <h2 id="Navigate">
                    Navigate the data from home page
                </h2>

                <p class="align-justify">
                    The OmicsDI home page provides different blocks to navigate through the datasets, some of them are: 2D
                    WordCloud; the species/organism/diseases bubble chart, repo/omics pie chart, Latest datasets, Most
                    accessed datasets, Datasets per year. All the charts allow the user to search the data using the
                    specific attribute. These boxes also act as an statistic component of the resource : for example the
                    pie chart shows how many datasets for each repository and omics the resource contains.
                </p>

                <p>
                    <img src="img/help/1.png" class="center">
                </p>


                <p class="align-justify">
                    A <b>TagCloud or WordCloud</b> is a visual representation for metadata, typically used to depict
                    keyword metadata (tags) on datasets, or to visualize free form text. The WordCloud is build using
                    the more frequently words for every database/repository. The OmicsDI <a>WordCloud</a> can be consider as
                    a two dimensional term representation where the user can select the database and the field they want
                    to look for: description vs database. The user can click the highlight word in the wordcloud to
                    search for this term in the resource.
                </p>


                <p class="align-justify">
                    The <b>bubble chart block</b> allows the users to navigate the data using three main categories:
                    Tissues, Organisms, and Diseases. The user can click in the bubble and it will be redirected to the
                    search using the clicked term.
                </p>


                <p class="align-justify">
                    The <b>Repo/Omics pie chart</b> and the <b>Omics vs Year bar chart</b> allow the users navigate the
                    data using the omics categories (metabolomics, transcriptomics, proteomics and genomics). The user can click a bar or
                    the pie and it will be redirected to the search using the clicked term.
                </p>


                <p class="align-justify">
                    The <b>Latest Datasets</b> and <b>Most accessed datasets</b> blocks provide a list of the datasets
                    by the tow categories.
                </p>


                <h2 id="Searchingdatasets">
                    Searching datasets
                </h2>

                <p class="align-justify">
                    The main search box in OmicsDI allows the user to search datasets using different keywords. The main
                    search redirects the user to the browser page where the user can see the results of the search (see
                    section 3.3).
                </p>

                <p>
                    <img src="img/help/2.png" class="center">
                </p>

                <p class="align-justify">
                    <code>
                        The OmicsDI search page is better that most of the partners (proteomeXchange, metabolomeXchange)
                        searching the data because the data is also indexed using cross-references to other databases.
                    </code>
                </p>

                <h3 id="Searchingusingpublication">
                    Searching using publication details
                </h3>


                <p class="align-justify">
                    The user can use the PubMed identifier, Title, Authors or even terms from the publication abstract.
                </p>


                <p>
                    <img src="img/help/3.png" class="center">
                </p>


                <h3 id="Searchingusingbiological">
                    Searching using biological Evidences
                </h3>


                <p class="align-justify">
                    The search box allows the end-users to search data using biological evidences such as the list of
                    the proteins identified in the proteomics experiment or the metabolite reported in the metabolomics
                    experiment. For example if the user search for 3-methyl-2-oxobutanoic in the resource it will found
                    one dataset in Metaboligths and five in Metabolome workbench that identified the current molecule.
                </p>


                <p>
                    <img src="img/help/4.png" class="center">
                </p>


                <h2 id="RefiningtheSearchresults">
                    Refining the Search results
                </h2>

                <p class="align-justify">
                    The search results can be filter or refine using different categories, filters or terms. The OmicsDI web
                    application supports at the moment nine different refinements: (1) omics type, (2)
                    repository/database, (3) organisms, (4) tissue, diseases, (5) modifications (proteomics), (6)
                    instruments and platforms, (7)publication data, (8) technology type.
                </p>


                <p>
                    <img src="img/help/5.png" class="center">
                </p>


                <p class="align-justify">
                    The refine filters works in combination, if the user set two filters they will act at the same time.
                </p>


                <h1 id="DatasetView">
                    Dataset View
                </h1>

                <p class="align-justify">
                    The dataset View shows the information for every dataset in the resource. It contains four main
                    information components: (1) the dataset description, (2) the sample/data protocol, (3) the
                    publication information, and (4) the similar datasets.
                </p>

                <p>
                    <img src="img/help/6.png" class="center">
                </p>


                <h1 id="OmicsDIArchitecture">
                    OmicsDI Architecture
                </h1>

                <p class="align-justify">
                    The OmicsDI project design is a modular architecture with three main components (Figure 1): (i)
                    Database/Repository Adapters, (ii) EBI Search Indexer, (iii) OmicsDI Application. In summary, the
                    Database/Repository Adapters provides all the libraries and readers to translate the original
                    repository/databases information into a universal and highly customizable XML file for each dataset
                    (see section 5.1). The EBI Search Indexer is a Lucene-based framework that enables to index all the
                    metadata and biological evidences for each dataset (see section 5.2). The OmicsDI application component
                    provides a MongoDB, web-services and web application for search and access the different datasets.
                </p>


                <p>
                    <img src="img/help/7.png" class="center">
                </p>


                <h2 id="DatabaseRepositoryAdapters">
                    Database/Repository Adapters
                </h2>

                <p class="align-justify">
                    The database/repository Adapter components are a set of readers and libraries that translate the
                    data from the original repositories to a common XML-based file. The OmicsDI-XML is a fully customizable
                    XML file used to represent the data from the different datasets, it contains as mandatory for each
                    dataset:
                <ul>
                    <li>Dataset ID</li>
                    <li>Dataset Title</li>
                    <li>Type of experiment</li>
                    <li>Publication date</li>
                    <li>Submitter details</li>
                </ul>
                A full description of the XML and the exporters can be found in <a href="https://github.com/BD2K-OmicsDI">Github </a>.
                If you need to add a new data resource, that is a new domain, get in touch first with the <a
                    href="omicsdi-support@ebi.ac.uk">OmicsDI team </a>.
                </p>

                <h2 id="EBISearchIndexer">
                    EBI Search Indexer
                </h2>

                <p>
                    In summary, the indexing pipeline:
                <ul>
                    <li>Based on Apache Lucene</li>
                    <li>An automatized indexing cycle takes place on daily basis.</li>
                    <li>Check every night, for every domain if new data is available.</li>
                    <li>The process benefits from the indexing parallelization.</li>
                    <li>All domains together: ~750GB, more than 300GB indexes</li>
                    <li>While the software releases are less frequent, the data to index is checked dail</li>
                </ul>
                </p>


            </div>
            <div class="col-md-3 col-lg-3 nav-float" style="position:sticky;top:6em">
                <nav id="affix-nav" class="sidebar  hidden-print visible-lg " role="complementary"
                     style="">
                    <ul style="width: 272px;" class="nav sidenav affix-top" data-spy="affix" id="affix-ul">
                        <li>
                            <a href="" ng-click="scrollTo('Overview')">Overview</a>
                        </li>
                        <li>
                            <a href="" ng-click="scrollTo('MajorPartner')">Major Partners</a>
                            <ul class="nav">
                                <li>
                                    <a href="" ng-click="scrollTo('ProteomeXchange')">ProteomeXchange</a>
                                </li>
                                <li><a href="" ng-click="scrollTo('MetabolomeXchange')">MetabolomeXchange</a></li>
                                <li><a href="" ng-click="scrollTo('EGA')">The European Genome-Phenome Archive</a></li>
                                <li><a href="" ng-click="scrollTo('CurrentDatabases')">Current Databases</a></li>
                            </ul>
                        </li>
                        <li>
                            <a href="" ng-click="scrollTo('OmicsDIwebapplication')">OmicsDI web application</a>
                            <ul class="nav">
                                <li><a href="" ng-click="scrollTo('Navigate')">Navigate the data from home page</a></li>
                                <li><a href="" ng-click="scrollTo('Searchingdatasets')">Searching datasets</a>
                                    <ul class="nav">
                                        <li><a href="" ng-click="scrollTo('Searchingusingpublication')">Searching using
                                            publication details</a></li>
                                        <li><a href="" ng-click="scrollTo('Searchingusingbiological')">Searching using
                                            biological Evidences</a></li>
                                    </ul>
                                </li>
                                <li><a href="" ng-click="scrollTo('RefiningtheSearchresults')">Refining the Search
                                    results</a></li>
                            </ul>
                        </li>
                        <li><a href="" ng-click="scrollTo('DatasetView')">Dataset View</a></li>
                        <li><a href="" ng-click="scrollTo('OmicsDIArchitecture')">OmicsDI Architecture</a>
                            <ul class="nav">
                                <li><a href="" ng-click="scrollTo('DatabaseRepositoryAdapters')">Database/Repository
                                    Adapters</a></li>
                                <li><a href="" ng-click="scrollTo('EBISearchIndexer')">EBI Search Indexer</a></li>
                            </ul>
                        </li>
                    </ul>
                </nav>
            </div>
        </div>
    </div>
    <!--container-->

</div>
<!--wrapper-->
